Endorser.md

```diff
@@ -0,0 +1,61 @@
+# Transaction Endorser Support
+
+Note that the ACA-Py transaciton support is in the process of code refactor and cleanup.  The following documents the current state, but is subject to change.
```

Comment:
Do you anticipate the API (e.g. Admin API and Startup Parameters) changing, or are those more or less set?

Reply:
I don't anticipate any API changes but can't guarantee we won't need any

Reply:
Typo "transaction"